homeassistant/components/ecobee/repairs.py

```diff
@@ -0,0 +1,37 @@
+"""Repairs support for Ecobee."""
+
+from __future__ import annotations
+
+from homeassistant.components.notify import DOMAIN as CLIMATE_DOMAIN
+from homeassistant.components.repairs import RepairsFlow
+from homeassistant.components.repairs.issue_handler import ConfirmRepairFlow
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import issue_registry as ir
+
+from .const import DOMAIN
+
+
+@callback
+def migrate_aux_heat_issue(hass: HomeAssistant) -> None:
+    """Ensure an issue is registered."""
+    ir.async_create_issue(
+        hass,
+        DOMAIN,
+        "migrate_aux_heat",
+        breaks_in_ha_version="2024.10.0",
+        issue_domain=CLIMATE_DOMAIN,
+        is_fixable=True,
+        is_persistent=True,
+        translation_key="migrate_aux_heat",
+        severity=ir.IssueSeverity.WARNING,
+    )
```

Comment:
Move this to `__init__.py`, there's no need to implement a repair platform since we don't implement a custom fix flow

Reply:
What do you think about removing the migrate_aux_heat_issue callback and just directly calling async_create_issue() directly in the two aux_heat locations that trigger the issue? That feels like it may be a bit cleaner with all the rest of that code going away.

Reply:
I've updated the PR to call async_create_issue() in the deprecated methods. Let me know if there is a reason that approach would be a problem.